packages/design-system/src/components/Tooltip/Tooltip.jsx

```diff
@@ -207,8 +239,25 @@ export class Tooltip extends React.Component {
   }
 
   render() {
+    const eventHandlers = this.props.dialog
+      ? {}
+      : {
+          onMouseEnter: () => {
+            if (!this.state.isMobile) {
+              this.setState({ isHover: true });
+              this.setTooltipActive(true);
+            }
+          },
+          onMouseLeave: () => {
+            if (!this.state.isMobile) {
+              this.setState({ isHover: false });
+              this.setTooltipActive(false);
+            }
+          },
+        };
+
     return (
-      <div>
+      <div {...eventHandlers}>
```

Comment:
Yes I think we can update this to a span

Reply:
Just to clarify though, the `triggerComponent` refers to [this element here](https://github.com/CMSgov/design-system/pull/947/files/ed3c14df370d2a875bae7806258fa599bb12b8c9#diff-1544310ec6e3788354a57d3ee54286c99bf4489ee0dee278fa7b6f8cd0e9dab6R155), which is what actually contains the glossary text or icon. That element should be a `button`, `a`, or rarely an `input` for accessibility requirements. The div you are linking to here though is just a container for the trigger component and the actual tooltip content, so it's easy to update to a `span`.

Reply:
Yup! Sorry, I realize my note in the review could have been confusing. Changing this line in particular to a `<span>` is separate from the `triggerComponent`.

I was talking about using an inline element for the `triggerComponent` to avoid this issue:
![Selection_160](https://user-images.githubusercontent.com/3094090/109839489-c61a5b80-7c0c-11eb-82ca-89a7711dc8e9.png)

Is there a way to make a `<button>` `triggerComponent` behave more like this?
![Selection_161](https://user-images.githubusercontent.com/3094090/109839609-e3e7c080-7c0c-11eb-9a31-d5389638c9a9.png)
(Screenshot is of a `<span>`.)

Reply:
Hm, I'm thinking about this a bit more. I got this to work when using `triggerComponent="a"`, and when I place the tooltip inline content, something like this:
```
<p className="ds-u-margin--0 ds-u-margin-right--1">
    {"Tooltip with "}
    <Tooltip triggerComponent="a" triggerContent="inline text trigger" triggerClassName="ds-c-tooltip__trigger-link">
        <span className="ds-u-margin--0">Tooltip trigger is styled with dashed underline</span>
    </Tooltip>
</p>
```

However, this isn't going to work because this places the tooltip content next to the trigger (for accessibility reasons), and you cant put `div`s and other HTML elements inside a `p` element. I'm going to have to think about this a bit more and handle this in an another PR

Reply:
Tagging @davidakennedy for visibility